### ISSUE TITLE
[Feature Request]: Ability to move captions

### ISSUE THREAD

**Issue by spaceghost24** [FreeTubeApp/FreeTube#5072]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Currently captions are only available in one location (centered at the bottom of the screen), and cannot be moved. This isn't an ideal location for many videos, as in-video text is often placed there (unfortunately), and since the progress bar pops up there as well when the video is paused, enjoying videos with text tends to involve a lot of turning the captions on and off and jumping back and forth to watch everything twice. On youtube I'm used to dragging the captions to the upper left-hand corner for videos like these, but I can't do that in freetube at the moment.

Making the captions smaller helps a bit, but 1) the setting is broken (the percentage seems to be a percentage of the current size at which they're rendered rather than of their default size, meaning that if you jump back and forth between two captions at 75% they'll just keep getting smaller and smaller until you reload the page) and 2) doesn't work for every situation.

### Proposed Solution

The simplest solution would probably be to have a checkbox in the captions setting to move the captions to the top of the video instead of the bottom of it, which would resolve most conflicts with text on-screen.

### Alternatives Considered

An option that lets you specify a custom location for the captions (rather than just top/bottom) would also work fine and give users more options, but would have to be implemented in a usable way -- my first thought was xy coordinates, but I think that would be confusing to use.

If hard-coded locations are chosen, having more options than just top/bottom would be nice (as I said, I'm partial to the top left corner) but I don't think it's necessary to make the captions more usable.

Being able to drag the captions around the screen the way you can on youtube would be ideal (it would mean you don't have to tinker with the settings menu for every video), but I don't know how difficult that would be to implement.

### Issue Labels

improvement to existing feature

**Comment by aneverhopefulhelper** [comment]
I would very much appreciate this feature too. It frustrates me when some elements are blocked by the subtitles and I can't quickly and easily move them out of the way.

**Comment by ChocoChopin** [comment]
It'd be nice to have _any_ options for captions at all, it seems we're only able to toggle them on and off. YouTube's own player offers plenty of caption styling options, we ought to have access to those at the very least.

**Comment by ozrendev** [comment]
This should be solved by #9723
